src/Symfony/Bridge/PhpUnit/bin/simple-phpunit

```diff
@@ -72,6 +72,11 @@ if (false === $COMPOSER = \getenv('COMPOSER_BINARY')) {
         ? (file_get_contents($COMPOSER, false, null, 0, 18) === '#!/usr/bin/env php' ? $PHP : '').' '.escapeshellarg($COMPOSER) // detect shell wrappers by looking at the shebang
         : 'composer';
 }
+if (in_array('never', $argv, true) || in_array('--colors=never', $argv, true)) {
```

Comment:
I'm not sure about the first check for `never`. Is this supposed to detect `--color never`?
Is yes, it should also check for `--color` before.
(same for "always" below)

Reply:
Yes, it was supposed to detect `--colors never`. I have added the checks for the option name for this call syntax